model/Index/Service/SyncResultIndexer.php

```diff
@@ -84,6 +84,9 @@ private function getSearch(): SearchInterface
 
     private function getIndexerService(): IndexService
     {
-        return $this->getServiceLocator()->get(IndexService::SERVICE_ID);
+        $service = $this->getServiceLocator()->get(IndexService::SERVICE_ID);
```

Comment:
This is actually a bug we have now and some indexation was not working. So it is not directly related to this task, but in order to index the data properly I needed to fix it.